tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -26,10 +27,16 @@ ${MS_TABLE_STATUS_FAILURE}=    //span[contains(@class,"pf-c-icon__content")][con
 *** Keywords ***
 Open Model Serving Home Page
     [Documentation]    Verifies submenu Settings > "Model Serving" is visible and navigates to page
-    Page Should Contain    Model Serving
-    Click Link      Model Serving
-    Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
-    Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
+    SeleniumLibrary.Page Should Contain    Model Serving
+    SeleniumLibrary.Click Link      Model Serving
+    Sleep    1s
+    ${loading_error} =    Run Keyword And Return Status    SeleniumLibrary.Page Should Contain    Problem loading model serving page
+    IF    ${loading_error}
+        Sleep    5s
+        SeleniumLibrary.Reload Page
+    END
```

Comment:
isn't the library search order handled in the Suite Setup of the test suite which is calling this kw? I believe it shouldn't be necessary to do so in the kw code

Reply:
I am trying to specify the library everywhere so that the search order doesn't matter

Reply:
hmm I personally don't like that because it make reading more difficult and add can be solved by one line only. We could talk in the automation meeting so we find an agreement otherwise we'll have some places where it is used and some places where it is not

Reply:
agreed. 
I don't like the search order approach because you won't ever be sure what is actually being run when the test runs by just reading the code.